Classes/Aspects/ContentCacheAspect.php

```diff
@@ -3,46 +3,31 @@
 
 namespace Flowpack\Varnish\Aspects;
 
-use Flowpack\Varnish\Service\CacheTagService;
 use Flowpack\Varnish\Service\VarnishBanService;
 use Neos\Flow\Annotations as Flow;
 use Neos\Flow\Aop\JoinPointInterface;
 use Neos\Flow\Configuration\Exception\InvalidConfigurationException;
 use Neos\Flow\Log\Utility\LogEnvironment;
 use Neos\Flow\Mvc\Exception\StopActionException;
+use Neos\Fusion\Core\Runtime;
 use Neos\Fusion\Exception;
 use Neos\Fusion\Exception\RuntimeException;
 use Neos\Utility\Exception\PropertyNotAccessibleException;
 use Neos\Utility\ObjectAccess;
-use Neos\Fusion\Core\Runtime;
 use Psr\Log\LoggerInterface;
 
 /**
  * Advice the RuntimeContentCache to check for uncached segments that should prevent caching
- *
- * @Flow\Aspect
- * @Flow\Scope("singleton")
  */
+#[Flow\Aspect]
+#[Flow\Scope("singleton")]
 class ContentCacheAspect
 {
+    #[Flow\Inject]
+    protected LoggerInterface $logger;
 
-    /**
-     * @Flow\Inject
-     * @var LoggerInterface
-     */
-    protected $logger;
-
-    /**
-     * @Flow\Inject
-     * @var VarnishBanService
-     */
-    protected $varnishBanService;
-
-    /**
-     * @Flow\Inject
-     * @var CacheTagService
-     */
-    protected $cacheTagService;
+    #[Flow\Inject]
+    protected VarnishBanService $varnishBanService;
```

Comment:
just a FYI: This changes the behavior slightly because previously the injections were _lazy_